bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/EntryLogger.java

```diff
@@ -1140,24 +1140,15 @@ public void shutdown() {
         entryLoggerAllocator.stop();
     }
 
-    static void closeFileChannel(BufferedChannelBase channel) throws IOException {
-        if (null == channel) {
-            return;
-        }
-
-        FileChannel fileChannel = channel.getFileChannel();
-        if (null != fileChannel) {
-            fileChannel.close();
+    static void closeFileChannel(BufferedChannel channel) throws IOException {
+        if (channel != null) {
+            channel.close();
         }
     }
 
-    static void forceCloseFileChannel(BufferedChannelBase channel) {
-        if (null == channel) {
-            return;
-        }
-        FileChannel fileChannel = channel.getFileChannel();
-        if (null != fileChannel) {
-            IOUtils.close(LOG, fileChannel);
+    static void forceCloseFileChannel(BufferedChannel channel) {
```

Comment:
@merlimat I don't see the need for this method. The caller can directly call "IOUtils.close(LOG, channel)", name of this method 'forceCloseFileChannel' is misleading and also it makes no sense to have this static method in EntryLogger class where it is just passing call to other static method.